pkg/cmd/enclave_secrets.go

```diff
@@ -167,7 +167,8 @@ func init() {
 	enclaveSecretsDownloadCmd.Flags().Bool("no-fallback", false, "disable reading and writing the fallback file")
 	enclaveSecretsDownloadCmd.Flags().String("fallback-passphrase", "", "passphrase to use for encrypting the fallback file. by default the passphrase is computed using your current configuration.")
 	enclaveSecretsDownloadCmd.Flags().Bool("fallback-readonly", false, "disable modifying the fallback file. secrets can still be read from the file.")
-	enclaveSecretsDownloadCmd.Flags().Bool("fallback-only", false, "read all secrets directly from the fallback file, without contacting Doppler. secrets will not be updated. (implies --fallback-readonly)")
+	enclaveSecretsDownloadCmdFallbackOnly := enclaveSecretsDownloadCmd.Flags().Bool("fallback-only", false, "read all secrets directly from the fallback file, without contacting Doppler. secrets will not be updated. (implies --fallback-readonly)")
```

Comment:
The `enclave` commands are deprecated so we can leave these unchanged

Reply:
[Force push](https://github.com/DopplerHQ/cli/compare/94b7f94a728ad5db03e49b5b756539db4c071b82..157a031786abe66609876f7f02c99016905de19e) removes the changes to the `enclave secrets download` command. Tested and works as expected - `Flags().Changed()` doesn't complain that the flag wasn't defined for the command.